Dockerfile

```diff
@@ -1,35 +1,36 @@
-FROM registry.fedoraproject.org/f26-modular/boltron
+FROM {{ config.docker.from }}
```

Comment:
You probably want to use `{{ config.docker.registry }}/{{ config.docker.from }}`

Reply:
Also, `{{ config.docker.from }}` will give you `fedora:26`, which is something that you didn't have before. You may want to consider if that's what you want to do.